### DISCUSSION TITLE
fixed cities/products names

### DISCUSSION THREAD

**Discussion by dauom** [geeksblabla/blanat#186] [Q&A] [answered]
Hi,
I see that the top-ranked solution assumes that the city and product names are fixed. Is this list from the gen.py file?
If that's guaranteed to stay the same even for the final scoring of our solution, please confirm and maybe it's worth mentioning in the README.

**Comment by aboullaite** [comment, marked as answer]
yes, the list is fixed. we use the same script to generate data
